### ISSUE TITLE
tileIds are wacky

### ISSUE THREAD

**Issue by lanery** [hoogenboom-group/scripted-render-pipeline#1]
tileId is currently set by
```python
spec.tileId = f"{sequential_id}_{self.name}_{self.zvalue}"
```
which is maybe not the most intuitive. I think something like
```python
row = spec.layout.imageRow
col = spec.layout.imageCol
spec.tileId = f"t{sequential_id}_z{self.zvalue}_y{row}_x{col}"
```
would be better, but not possible without proper z indexing.

**Comment by thopp-tudelft** [comment]
I thought the tileid is chosen to determine the order in which it is rendered, and just has to be unique, the x and y coordinates are easily accessible elsewhere, are you worried that without them the tileId wouldn't be unique in some cases?

**Comment by lanery** [comment]
yes the key requirements are indeed uniqueness and correct chronological ordering within a stack, but then I think it's beneficial to have the tileId as immediately recognizable as possible within those constraints. it's true that the row and column information can be retrieved from the layout, but that's two additional queries that can be avoided.

**Comment by lanery** [comment]
Fixed with commit 777951d231822d467fcfa88b3f116ad94d52e0fc
